framework/Samplers/Sampler.py

```diff
@@ -297,14 +297,10 @@ def readSamplerInit(self,xmlNode):
       if child.tag == "samplerInit":
         self.initSeed = Distributions.randomIntegers(0,2**31,self)
         for childChild in child:
-          if childChild.tag == "limit":
```

Comment:
MonteCarlo.py  However, since Sampler.py in amIReadyToProvideAnInput has:
```
ready = True if self.counter < self.limit else False
```
I think the correct place to read limit is in Sampler.